packages/chrome/src/ChromeProvider/ChromeProvider.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { useLocation } from 'react-router-dom';
+import { useFlag } from '@unleash/proxy-client-react';
+
+import { ChromeContext } from '../ChromeContext';
+import chromeState from './chromeState';
+import { get, post } from '../utils/fetch';
+
+const getLastVisited = () => get<string[]>('/api/chrome-service/v1/last-visited');
+
+const useLastPageVisitedUploader = (providerState: ReturnType<typeof chromeState>, chromeBackendEnabled?: boolean) => {
+  const { pathname } = useLocation();
+  useEffect(() => {
+    if (chromeBackendEnabled) {
+      post<string[], { pathname: string; title: string }>('/api/chrome-service/v1/last-visited', {
```

Comment:
Can we make `'/api/chrome-service/v1/last-visited'` into a constant? Or even better use JS client?

Reply:
The client is too heavy (you know no tree shaking). But yeah I will add a constant.